packages/web/src/builders/package/package.impl.ts

```diff
@@ -83,6 +89,18 @@ export function run(
           updatePaths(dependencies, parsedTSConfig.compilerOptions.paths);
 
           const plugins = [
+            typescript({
+              check: true,
+              tsconfig: options.tsConfig,
+              tsconfigOverride: {
+                compilerOptions: {
+                  rootDir: options.entryRoot,
+                  allowJs: false,
+                  declaration: index === 0,
+                  paths: parsedTSConfig.compilerOptions.paths
+                }
+              }
+            }),
```

Comment:
we want to invoke the typescript compiler each time, so it can fail the build if something is wrong

Reply:
Make sense. I just wanna check to make sure the type error is only printed once rather than multiple times.

Reply:
I tested the build and it does output the same errors twice (e.g. type errors) if both rollup runs fail the same way. See my other comment for more on this.

https://github.com/nrwl/nx/pull/2524#discussion_r382776806